cubeviz/cubeviz.py

```diff
@@ -41,9 +40,9 @@ def main(argv=sys.argv):
     args = parser.parse_known_args(argv[1:])
 
     # Store the args for each ' --data-configs' found on the commandline
-    global global_data_configuration
-    global_data_configuration['data_configs'] = args[0].data_configs if args[0].data_configs else []
-    global_data_configuration['data_configs_show'] = args[0].data_configs_show
+    data_configuration = {}
+    data_configuration['data_configs'] = args[0].data_configs if args[0].data_configs else []
+    data_configuration['data_configs_show'] = args[0].data_configs_show
```

Comment:
Do you actually need the dictionary here, or can you just store in variables called ``data_configs`` and ``data_configs_show``?